fdbclient/NativeAPI.actor.cpp

```diff
@@ -3273,20 +3282,21 @@ ACTOR Future< StorageMetrics > waitStorageMetrics(
 	}
 }
 
-Future< StorageMetrics > Transaction::waitStorageMetrics(
```

Comment:
Just another moment when I wish c++ has some built-in `Either<L, R>` and case-matching support and... I guess I am wishing too much.